library/src/com/google/maps/android/kml/KmlGeometry.java

```diff
@@ -0,0 +1,24 @@
+package com.google.maps.android.kml;
+
+/**
+ * Represents a KML geometry object.
+ *
+ * @param <T> type of object that the coordinates are stored in
+ */
+public interface KmlGeometry<T> {
+
+    /**
+     * Gets the type of geometry
+     *
+     * @return type of geometry
+     */
+    public String getKmlGeometryType();
+
+    /**
+     * Gets the stored KML Geometry object
+     *
+     * @return geometry object
+     */
+    public T getKmlGeometryObject();
```

Comment:
All classes that implement this interface return coordinates except for multigeometry which returns an array of geometries